examples/features/driven_dynamics/efield-water/README.md

```diff
@@ -0,0 +1,157 @@
+    \frac{\partial P_y}{\partial R^1_y} &
+    \frac{\partial P_z}{\partial R^1_y} \\
+    \frac{\partial P_x}{\partial R^1_z} &  
+    \frac{\partial P_y}{\partial R^1_z} &
+    \frac{\partial P_z}{\partial R^1_z} \\
+    \frac{\partial P_x}{\partial R^2_x} &  
+    \frac{\partial P_y}{\partial R^2_x} &
+    \frac{\partial P_z}{\partial R^2_x} \\ 
+    \vdots & \vdots & \vdots \\
+    \frac{\partial P_x}{\partial R^N_z} &  
+    \frac{\partial P_y}{\partial R^N_z} &
+    \frac{\partial P_z}{\partial R^N_z} \\    
+\end{bmatrix}
+$$
+- by setting their values "manually" in `input.xml`:
+```xml
+<motion mode='dynamics'> 
+    <dynamics mode='eda-nve'>                  <!-- Electric Dipole Approximation (EDA) on top of a NVE simulation -->
+        <timestep units='femtosecond'> 1 </timestep>
+        <efield>
+            <amp> [ 0 , 5e-2 , 0 ] </amp>             <!-- amplitude of the electric field pulse in a.u. -->
+            <freq units="thz"> 115 </freq>            <!-- frequency of the electric field pulse in THz -->
+            <peak units="picosecond"> 0.2 </peak>     <!-- peak of the electric field pulse -->
+            <sigma units="picosecond"> 0.05 </sigma>  <!-- width (standard deviation of the gaussian) of the electric field pulse -->
+        </efield>
+        <bec mode="file"> 
+        [    0.7221015439,	-0.0000028201,	-0.0000000012,
+            -0.0000028201,	-0.4070007918,	 0.0001345078,
+             0.0000000000,	-0.0000003644,	-0.6096787984,
+             0.3610505794,	 0.0000014101,	 0.0000003858,
+             0.0000014101,	 0.2035008826,	-0.0431014831,
+             0.0000006580,	-0.0735216582,	 0.3047408429,
+             0.3610509645,	 0.0000014101,	-0.0000003846,
+             0.0000014101,	 0.2034999092,	 0.0429669754,
+            -0.0000006580,	 0.0735220226,	 0.3049379554 ]  
+        </bec>
+    </dynamics>
+</motion>
+```
+where the array that has to be provided in `input.xml` is just the "flattened" version of the matrix contained in `bec.txt`.
+
+#### Energy and forces
+In this example the energy and forces of the isolated water molecule are provided by the ```pswater``` model, but you are free to use your favorite DFT code.
+ 
+## How to run
+Let's assume that you have the ```i-PI``` code in th folder ```i_pi_path```.
+Let's source all the necessary ```i-PI``` enviromental variable:
+```bash
+source ${i_pi_path}/env.sh
+```
+
+Let's run ```i-PI```:
+```bash
+i-pi input.xml  > i-pi.out &
+```
+
+Let's run the driver (using a UNIX socket) for the energy and forces:
+```bash
+i-pi-driver -u -a driver -m pswater # > pswater.out &
+```
+In case you want to use an INET socket, modify ```input.xml``` as described by the comments therein, and run the driver with:
+```bash
+i-pi-driver -a ipaddress -p 27683 -m pswater # > pswater.out &
+```
+where ```ipaddress``` is your ```ip``` address and the port number (```-p```) should respect the criteria provided by the ```i-PI``` manual.
```

Comment:
I don't think these belong here. We don't want to repeat 50 times how to run i-PI, focus on the part that is efield specific

Reply:
I removed this part from the `README.md` file